nullaway/src/main/java/com/uber/nullaway/handlers/GrpcHandler.java

```diff
@@ -59,10 +67,10 @@ public void onMatchTopLevelClass(
       NullAway analysis, ClassTree tree, VisitorState state, Symbol.ClassSymbol classSymbol) {
     if (grpcMetadataType == null || grpcKeyType == null) {
       grpcMetadataType =
-          Optional.ofNullable(state.getTypeFromString(GRPC_METADATA_TNAME))
+          Optional.ofNullable(GRPC_METADATA_TYPE_SUPPLIER.get(state))
```

Comment:
Since this code only ever runs once, we could suppress the warning from MemoizeConstantVisitorStateLookups rather than adding the Supplier